pyproject.toml

```diff
@@ -42,6 +42,8 @@ ulid = "^1.1"
 # TODO: Remove this arbitrary python constraint once `sqlalchemy-bigquery` has done so.
 sqlalchemy-bigquery = { version = "1.9.0", python = "<3.13" }
 
+airbyte-source-declarative-manifest = { version = "0.79.1", python = ">=3.9,<3.12" }
```

Comment:
Note: We'd need to keep these versions aligned, but there's no other net new dependencies as a result of the addition. 👍

Reply:
Removed! 🎉